### PR TITLE
Use explicit stopping state when stopping triggered by Ankaios

### PR DESCRIPTION
**Pull request by krucod3**
This fixes an issue introduced by the changes for #48 

# Definition of Done

The PR shall be merged only if all items mentioned in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been followed. In case an item is not applicable as described, please provide a short explanation in the description.

- [ ] All steps in [CONTRIBUTING.md](https://github.com/eclipse-ankaios/ankaios/blob/main/CONTRIBUTING.md#how-to-contribute) have been handled

### CODE REVIEW

**Comment by inf17101** [comment]
@krucod3: I think the state diagram must be adapted, right?

**Review by inf17101 — APPROVED** [review]
Optional: When I walk along the arrows as an external person without any prior knowledge, I often turn my eye in the wrong direction because the direction is often lost due to the overlap.

LGTM 👍

**Comment by krucod3** [comment]
@inf17101, yeah ... I had the same issues. At the end I decides it would be best to simplify it and only show the transitions between the major states. A new version is available, please take a look. Thanks!

File: common/doc/swdesign/drawio/state_workload_execution_major_states.drawio.svg
[the remark this answers is not in the document]
Comment (krucod3) [reply common/doc/swdesign/drawio/state_workload_execution_major_states.drawio.svg:1]: Fixed.

**Pull request merged**